Community/kavSimulations/config/KavSim Pedestal Board Config v1-0-0.mfmc

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <ModuleName>Kav Mega Pedesta</ModuleName>
```

Comment:
Didn't we even increase the length of the name to 40 characters?

Otherwise, drop the Mega from the name and add the `ModuleType`.

Reply:
ModuleName changed and ModuleType added

Yes, we agreed to increase the length to 40 characters. The FW can handle up to 48, but we wanted to be on the safe side